rel/app/env.bat.eex

```diff
@@ -2,8 +2,10 @@ if exist "!USERPROFILE!\.livebookdesktop.bat" (
   call "!USERPROFILE!\.livebookdesktop.bat"
 )
 
+for /f "skip=1" %%X in ('wmic os get localdatetime') do if not defined TIMESTAMP set TIMESTAMP=%%X
+
 if defined LIVEBOOK_NODE set RELEASE_NODE="!LIVEBOOK_NODE!"
-if not defined RELEASE_NODE set RELEASE_NODE=livebook_server
+if not defined RELEASE_NODE set RELEASE_NODE="livebook-app-!TIMESTAMP:~8,13!-!RANDOM!"
```

Comment:
this generates a hostname like this:

```
livebook-app-115832.309000-24850
```

which is rejected by net_kernel:

```
    args: []
    label: {error_logger,info_msg}
    format: "Invalid node name!\nPlease check your configuration\n"
2024-03-01 11:58:32.777000 crash_report
    initial_call: {net_kernel,init,['Argument__1']}
```

It seems the dot is the issue. WDYT about dropping microseconds?

```suggestion
if not defined RELEASE_NODE set RELEASE_NODE="livebook-app-!TIMESTAMP:~8,6!-!RANDOM!"
```

```
livebook-app-120212-25569
```